provider/clusterrosaclassic/cluster_rosa_classic_state.go

```diff
@@ -99,3 +105,34 @@ type PrivateHostedZone struct {
 	ID      types.String `tfsdk:"id"`
 	RoleARN types.String `tfsdk:"role_arn"`
 }
+
+func flattenAdminCredentials(username string, password string) types.Object {
```

Comment:
Please add another function for ObjectNull, maybe something like that 
```
func flattenAdminCredentials() types.Object {
 return types.ObjectNull(map[string]attr.Type{
		"username": types.StringType,
		"password": types.StringType,
	})
}

```

Reply:
Or alternatively it can be like:
```
func adminCredentialsNull() types.Object {
  return flattenAdminCredentials("", "")
}
```
Up to you :-)

Reply:
I select the 2nd option for clean code
thanks